target/linux/ramips/dts/WIZFI630S.dts

```diff
@@ -0,0 +1,196 @@
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "wiznet,wizfi630s", "mediatek,mt7628an-soc";
+	model = "WIZnet WizFi630S";
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x8000000>;
+	};
+
+        aliases {
+                led-boot = &led_run;
+                led-failsafe = &led_run;
+                led-running = &led_run;
+                led-upgrade = &led_run;
+        };
+
+	gpio-leds {
+		compatible = "gpio-leds";
+		
+		led_run: run {
+			label = "wizfi630s:green:run";
+			gpios = <&gpio0 4 GPIO_ACTIVE_LOW>;
+		};
+
+		ledwps {
+			label = "wizfi630s:green:wps";
+			gpios = <&gpio1 10 GPIO_ACTIVE_LOW>; 
+		};
+
+		leduart1 {
+			label = "wizfi630s:green:uart1";
+			gpios = <&gpio0 0 GPIO_ACTIVE_LOW>;
+		};
+
+		leduart2 {
+			label = "wizfi630s:green:uart2";
+			gpios = <&gpio0 3 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	gpio-keys-polled {
```

Comment:
@7tobias  Could you please change this to generic name `keys` as suggested by [DTS Checklist](https://openwrt.org/submitting-patches#dts_checklist)
>  5. The name of a node should reflect the function of the device and not its model. Examples for generic node names can be found in Section 2.2.2 Generic Names Recommendation